src/main/java/ua/danit/final_project/controllers/exceptions/GeneralExceptionHandler.java

```diff
@@ -57,4 +63,10 @@
     return ResponseEntity.badRequest()
         .body(new ErrorResponse("Bad request.", 400));
   }
+
+  @ExceptionHandler({MaxUploadSizeExceededException.class, FileUploadBase.FileSizeLimitExceededException.class})
+  public ResponseEntity<?> uploadMaxSizeException() {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove usage of generic wildcard type. ([squid:S1452](https://sonarcloud.io/organizations/final-project/rules#rule_key=squid%3AS1452))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=final-project&pullRequest=153&issues=AWaluGC1rChX7_AlV7Y3&open=AWaluGC1rChX7_AlV7Y3)

[ISSUE_KEY]: AWaluGC1rChX7_AlV7Y3